xml_inscriptions/transcribed/MI.AA.UM.KM.L.1052.xml

```diff
@@ -110,20 +118,18 @@
                      </expan>
                   </name> 
                   <name type="gentilicium" key="Faenius">Faenius</name>, <expan>
-                     <abbr>L</abbr><ex>uciorum duorum</ex>
+                     <abbr>LL</abbr><ex>uciorum</ex>
```

Comment:
I made the change myself but one thing to note: when you have these kind of duplicated letters where one of them is indicating the plural abbreviation, mark one of them with `<am>` (abbreviation mark), which will prevent them from displaying in your transcription.